boofuzz/sessions.py

```diff
@@ -10,6 +10,7 @@
 import time
 import traceback
 import zlib
+import socket
```

Comment:
Importing full `socket` seems a little overkill here.
I'd suggest using something like `from socket import error as socket_error` instead.

Reply:
It's a bit of a coding style question -- AFAIK there's no performance difference in this case. Either way it has to import socket, it's just a question of namespaces. I personally prefer the `import socket` style as `socket.error` in the code makes it very easy/consistent to tell where it came from.

Reply:
Either way is fine IMO, unless I'm wrong about the performance aspect.

Reply:
No problem I already have the change made and just need to rebase.